CHANGELOG.md

```diff
@@ -128,9 +128,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Initial release to PyPI
 
+<<<<<<< HEAD
```

Comment:
@jajreidy you might have had some trouble with squashing commits.  Markers like this shouldn't be here, and also the PR still contains 18 commits.  Could you have another try?  I can show you on a screen share if you want.